website/docs/r/project.html.markdown

```diff
@@ -83,6 +83,14 @@ The following arguments are supported:
 
 * `initialize_with_readme` - (Optional) Create master branch with first commit containing a README.md file.
 
+* `template_name` - (Optional) When used without use_custom_template, name of a built-in project template. When used with use_custom_template, name of a custom project template.
+
+* `template_project_id` - (Optional)  When used with use_custom_template, project ID of a custom project template. This is preferable to using template_name since template_name may be ambiguous (enterprise edition).
```

Comment:
I know this is straight from the Gitlab docs, but perhaps you could add a note in here about `template_name` and `template_project_id` being mutually exclusive.

Reply:
will check mutal exclusive first and will adapt

Reply:
Added a note